src/twisted/python/test/_deprecatetests.py.3only

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) Twisted Matrix Laboratories.
+# See LICENSE for details.
+
```

Comment:
(required)
This should be written up in the Python 3 porting guide, I think. Note that one thing learned during initial porting efforts was that splitting files into Python 2 / Python 3 caused a lot of problems (conflict related). Perhaps this pattern is only going to be used for new files, though, and so that experience is not relevant. Just mentioning it.

There is some code in Twisted that employs an alternate idiom, putting the source into a string and then exec()ing it when necessary. I think it would be good to employ one or the other of these - not both.